### DISCUSSION TITLE
Wolfi is going on tour! 🔎

### DISCUSSION THREAD

**Discussion by kirby-koo** [wolfi-dev/os#2238] [General]
🔎 For the first time ever, we're taking Wolfi on tour! We'll be doing demos, answering questions, live coding and of course gifting swag! The first two cities we will be going to are:

- [Dallas (6/20/23)](https://www.eventbrite.com/e/falco-on-tour-dallas-special-guest-wolfi-tickets-624750544697) 
- [Denver (6/22/23)](https://www.eventbrite.com/e/falco-on-tour-denver-special-guest-wolfi-tickets-627307342147)

Please click the cities to register, and learn more about Wolfi with @eddiezane! 
Any other cities you'd like us to go to? Comment below 🤠

**Comment by JonZeolla** [comment]
Pittsburgh, PA 😁

**Reply by omissis** [reply]
Milan, Italy 🇮🇹

**Comment by developer-guy** [comment]
Istanbul, Turkiye 🇹🇷

**Comment by EzzOps** [comment]
Alexandria, Egypt
